cdk_typescript/src/detectors/typescript-cdk-emrs-3-access-logging/typescript-cdk-emrs-3-access-logging_compliant.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+// {fact rule=typescript-cdk-emrs-3-access-logging@v1.0 defects=0}
```

Comment:
could you make sure the rules version is same as the rule registry, we are using the `typescript-cdk-emrs-3-access-logging:0.1`